src/interfaces/strategy.ts

```diff
@@ -151,7 +151,8 @@ export class StrategyInterface<T extends ChainId> extends ServiceInterface<T> {
         const metadata: StrategyMetadata = {
           address: strategy.address,
           name: metadatum?.name || strategy.name || "Strategy",
-          description: metadatum?.description ?? "I don't have a description for this strategy yet",
+          description:
+            metadatum?.localization[this.ctx.locale]?.description ?? "I don't have a description for this strategy yet",
```

Comment:
fallback to the english description first instead of `"I don't have a description for this strategy yet"`? Since that's in english anyway

Reply:
Agreed if we are certain that English is always available.

Otherwise, I'd say "No description available" or something more generic.

Reply:
metadatum.description should always be available and in english, so I'm confident we can fallback to that, as the PR has been updated to do

Reply:
fixed on d53eb2fc1b404ad58a1f6df757b73691fbf57a05